manifests/b/badpanda/GifYourGame/5.2.2/badpanda.GifYourGame.locale.en-US.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: badpanda.GifYourGame
+PackageVersion: 5.2.2
+PackageLocale: en-US
+Publisher: Bad Panda, Inc.
+PublisherUrl: https://www.gifyourgame.com
+PublisherSupportUrl: https://discord.gg/KwhjMR6
+Author: BadPanda
+PackageName: Gif Your Game
+License: Copyright © 2021 Bad Panda, Inc.
+ShortDescription: The number #1 tool for clipping your favorite games. Featuring Zero Impact Clipping, Screen Clipping and start & stop based recording.

```

```suggestion
ShortDescription: "The number #1 tool for clipping your favorite games. Featuring Zero Impact Clipping, Screen Clipping and start & stop based recording."
```
Oops, one more change since `#` is a comment in YAML.